experiment/bootstrap/repos.go

```diff
@@ -57,7 +57,7 @@ func (r *Repo) GitBasePath(ssh bool) string {
 
 // PullNumbers converts a Pull's list string into a slice of PR number strings
 // NOTE: this assumes that if there are pull requests specified that Repo.Pull
-// looks something like: `master:hash,prNum:hash,prNum:hash,...`
+// looks something like: `master:hash,prNum:hash,...`
```

Comment:
Looking at the code, I think the comment is correct (?)... note that it returns a `[]string`